src/engine/ruleset/schemas/ecs_types.json

```diff
@@ -1,13 +1,13 @@
 [
-  "date",
   "ip",
-  "object",
-  "text",
   "long",
-  "boolean",
+  "object",
   "geo_point",
-  "scaled_float",
   "keyword",
   "nested",
+  "scaled_float",
+  "text",
+  "boolean",
+  "date",
   "float"
 ]
```

Comment:
Inserts a line break at the end of the file.

Reply:
Done